### PR TITLE
adding gtm to base template

### PR DESCRIPTION
**Pull request by calellowitz**
@nickpell @millerdev

### CODE REVIEW

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html [hunk 1]
```diff
@@ -228,6 +236,10 @@
         {% endif %}
     </head>
     <body>
+        <!-- Google Tag Manager (noscript) -->
+        <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-52MT7FS"
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/base.html]: Maybe not necessary, given how dysfunctional HQ is if you don't have scripting enabled.

**Review by orangejenny — APPROVED** [review]

**Review by millerdev — COMMENTED** [review]
If I'm reading the description of GTM correctly, it seems like we could eliminate some code here too. Possibly https://github.com/dimagi/commcare-hq/blob/986584382da8efa1eb43a32a01205e14bf5e531e/corehq/apps/hqwebapp/templates/hqwebapp/includes/analytics_all.html#L6
Is that part of the plan?

**Comment by calellowitz** [comment]
@millerdev Where are you reading that? As far as I know that is not part of the plan. This is being added for some sales test that the marketing team is running. But my understanding was that this supplements google analytics, not replaces it

**Comment by millerdev** [comment]
[Google Tag Manager overview](https://support.google.com/tagmanager/answer/6102821?hl=en)

> Google Tag Manager is a tag management system that allows you to quickly and easily update tags and code snippets on your website or mobile app, such as those intended for traffic analysis and marketing optimization. You can add and update AdWords, Google Analytics, Firebase Analytics, Floodlight, and 3rd party or custom tags from the Tag Manager user interface instead of editing site code.

**Comment by calellowitz** [comment]
Yeah, I read that as you can use google tag manager to do "tags" (whatever that is) in google analytics and other analytics platforms. It adds that new feature, but does not replace or seek to replicate any of the other functionality GA provides.

**Comment by millerdev** [comment]
> "tags" (whatever that is)

I was, perhaps incorrectly, assuming this means GTM allows you to manage (that is, a GTM user could add or remove) various other analytics snippets on the page. Normally these things are added in the form of a `<script>` "tag", which is what made me think that. In this case it sounds like we're at least not using it in that capacity yet, so fine as is.

**Comment by calellowitz** [comment]
Ah. I totally misinterpreted that. I think you may be correct. I was picturing tags as labels you attach to events. I can talk to carrie about moving in that direction longer term but its not on the roadmap for now.

**Pull request merged**